tools/ci_build/github/azure-pipelines/stages/nuget-combine-cuda-stage.yml

```diff
@@ -189,8 +189,25 @@ stages:
           parameters:
             PackageType: 'nuget'
             PackagePath: '$(Build.ArtifactStagingDirectory)'
-            PackageName: 'Microsoft.ML.OnnxRuntime.*nupkg'
             PlatformsSupported: 'win-x64,linux-x64'
+            # 1* stands for version number. we use it to fileter Gpu.Windows and Gpu.Linux packages
+            PackageName: 'Microsoft.ML.OnnxRuntime.Gpu.1*nupkg'
```

Comment:
>1* [](start = 55, length = 2)

how does 1* work? does it still work for string like Microsoft.ML.OnnxRuntime.Gpu.2.0.0.dev.xxx.nupkg?

Reply:
I believe it would not.

Reply:
also found this
tools/nuget/validate_package.py:    is_gpu_package = bool("microsoft.ml.onnxruntime.gpu.1" in args.package_name.lower())

---
In reply to: [1446715440](https://github.com/microsoft/onnxruntime/pull/19054#discussion_r1446715440) [](ancestors = 1446715440)

Reply:
Let's fix it in a different PR. I copied it from existing implementations from Yi Zhang's PR. If we want to change it, we need to fix both places.